notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>One quick description either 1) markdown here or 2) inside the function as a comment (feel free to correct it as I might not fully understand the comment.) </p><p> </p><blockquote>A function to find non zero values and index for a given dataframe column. from the end of the input dataframe. It will return zero there if there is only zero for input. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>It returns zero when the velocity is close to zero, so the position can be taken at that specific velocity.</p>